dart/lib/src/sentry_options.dart

```diff
@@ -189,6 +190,25 @@ class SentryOptions {
   /// Whether to send personal identifiable information along with events
   bool sendDefaultPii = false;
 
+  /// Whether [SentryEvent] deduplication is enabled.
+  /// Can be further configured with [exceptionsToKeepForDeduplication].
+  /// Shoud be set to true if
+  /// [SentryHttpClient] is used to capture failed requests.
+  bool enableDeduplication = true;
+
+  int _exceptionsToKeepForDeduplication = 5;
+
+  /// Describes how many exceptions are kept to be checked for deduplication.
+  /// This should be a small positiv integer in order to keep deduplication
+  /// performant.
+  /// Is only in effect if [enableDeduplication] is set to true.
+  int get exceptionsToKeepForDeduplication => _exceptionsToKeepForDeduplication;
```

Comment:
tbh the name is a bit off for me, but I am not sure what to suggest either.
maybe @bruno-garcia or @philipphofmann ?

`maxDeduplicationQueueItems` looks better IMO

Reply:
Maybe `maxDeduplicationItems`. It's no really relevant for the user how exceptions are stored, isn't it?

Reply:
`maxDeduplicationItems` also makes sense to me. Do we even need to make this option public?

Reply:
No, not necessarily. The value of it is just a guess for what should work well. So that's why I made it configureable.

Reply:
other languages have a WeakRef list so it's not necessary having such an option.
here the approach is different, and depending on how you use the SDK, 5 items may be enough, or not.